Neos.Flow/Classes/Mvc/Routing/Router.php

```diff
@@ -267,9 +276,112 @@ protected function createRoutesFromConfiguration()
             }
             $this->routes[] = $route;
         }
+
         $this->routesCreated = true;
     }
 
+    /**
+     * Emits a signal when the router configuration has been loaded
+     *
+     * This signal can be used to add routes during runtime. In the slot make sure to receive the
+     * $routerConfiguration array by reference so you can alter it.
+     *
+     * @param array $routerConfiguration The router configuration
+     * @return void
+     * @Flow\Signal
+     */
+    protected function emitConfigurationLoaded(array &$routerConfiguration): void
```

Comment:
The public API is `RouterInterface`.. I wonder if we could have the signal there somehow.. But I doubt that's possible :-/